### ISSUE TITLE
Does SpringRoll support keyboard navigation?

### ISSUE THREAD

**Issue by samreid** [SpringRoll/SpringRoll#39]
An important accessibility feature is the ability to navigate using the keyboard (tab, space, enter).  Does SpringRoll provide this support?

**Comment by bigtimebuddy** [comment]
We created a SpringRoll plugin for dealing with the keyboard: https://github.com/SpringRoll/Keyboard. If you use another keyboard library or implement your own listeners, it's important to turn _on_ the following Application option:

```js
var app = new Application({
    keepFocus: true
});
```
This ensures that your Application's iframe is always remains focused while running with a Container.

**Comment by andrewstart** [comment]
To give a more specific answer - no, SpringRoll does not have built in support for keyboard navigation. SpringRoll does not provide a full UI suite, just a basic button class.
